src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        final int arraySize = (int) (Math.random() * 10) + 1;
+        Figure[] figures = new Figure[arraySize];
```

Comment:
create constant SIZE_OF_ARRAY for example 6
```suggestion
        Figure[] figures = new Figure[SIZE_OF_ARRAY];
```

Reply:
not fixed